process_jartic_simple.py

```
@@ -0,0 +1,163 @@
+            measurements_in_archive = 0
+            
+            with zipfile.ZipFile(archive, 'r') as main_zip:
+                prefecture_files = [f for f in main_zip.namelist() if f.endswith('.zip') and not f.startswith('__MACOSX')]
+                
+                with open(output_path, 'a', encoding='utf-8') as out_file:
+                    for pref_file in prefecture_files:
+                        try:
+                            prefecture = pref_file.replace('.zip', '').split('/')[-1]
+                            pref_data = main_zip.read(pref_file)
+                            
+                            import io
```

Pylint (3.3.7)

[convention] Import outside toplevel (io)

(C0415)